README.md

```diff
@@ -44,3 +44,15 @@ Linters order above is the preffered way to run and fix them one by one.
     or `set ...` for Windows.
 3. Type `flask run` command in terminal.
 4. Open http://127.0.0.1:5000 in your internet browser.
+
+
+
+### Run database container
+
+1. In terminal type in `docker-compose -f envs/dev/docker-compose.yml up --detach` command. (you need Docker and Docker Compose installed)
+
+
+
+### Apply migrations to database
+
+1. In terminal type in `alembic upgrade <revision id>`. (Revision ID may be found in forum123/alembic/versions/<revision>)
```

Comment:
I've found in the [docs](https://alembic.sqlalchemy.org/en/latest/tutorial.html#running-our-first-migration) that we can run `alembic upgrade head` if we want to apply all migrations up to the last one. So we can change command here to `alembic upgrade head`